test/integration/wyatt_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+class WyattTest < ActionDispatch::IntegrationTest
+  test "shows specific message when visiting '/wyatt'" do
+    get "/wyatt"
+    assert_response :success
```

Comment:
this is an assertion.

in this case we are asserting that the response after you make the GET request to /wyatt is a HTTP 200, or a success.